benchmark/Main.hs

```diff
@@ -0,0 +1,69 @@
+import           Criterion.Main
+import           Data.Hashable
+import qualified Data.HashSet       as HSet
+import           Data.List          (group, head, nub, sort)
+import qualified Data.List.NonEmpty as S
+import qualified Data.Set           as Set
+
+import           List               (hashNub, ordNub)
+
+main :: IO ()
```

Comment:
I think, now benchmarks are not tested in CI. Could you please modify `.travis.yml` file in a way how it's done here:

https://github.com/serokell/importify/blob/3c26c156306b6a95ea1e02344c785354cac59d98/.travis.yml#L31